components/resources/brave_components_strings.grd

```diff
@@ -105,60 +105,60 @@
-    <file path="strings/brave_components_resources_fil.xtb" lang="fil" />
-    <file path="strings/brave_components_resources_fr.xtb" lang="fr" />
-    <file path="strings/brave_components_resources_fr-CA.xtb" lang="fr-CA" />
-    <file path="strings/brave_components_resources_gl.xtb" lang="gl" />
-    <file path="strings/brave_components_resources_gu.xtb" lang="gu" />
-    <file path="strings/brave_components_resources_hi.xtb" lang="hi" />
-    <file path="strings/brave_components_resources_hr.xtb" lang="hr" />
-    <file path="strings/brave_components_resources_hu.xtb" lang="hu" />
-    <file path="strings/brave_components_resources_id.xtb" lang="id" />
-    <file path="strings/brave_components_resources_it.xtb" lang="it" />
-    <file path="strings/brave_components_resources_iw.xtb" lang="he" />
-    <file path="strings/brave_components_resources_ja.xtb" lang="ja" />
-    <file path="strings/brave_components_resources_kn.xtb" lang="kn" />
-    <file path="strings/brave_components_resources_ko.xtb" lang="ko" />
-    <file path="strings/brave_components_resources_lt.xtb" lang="lt" />
-    <file path="strings/brave_components_resources_lv.xtb" lang="lv" />
-    <file path="strings/brave_components_resources_ml.xtb" lang="ml" />
-    <file path="strings/brave_components_resources_mr.xtb" lang="mr" />
-    <file path="strings/brave_components_resources_ms.xtb" lang="ms" />
-    <file path="strings/brave_components_resources_nl.xtb" lang="nl" />
-    <file path="strings/brave_components_resources_no.xtb" lang="no" />
-    <file path="strings/brave_components_resources_pl.xtb" lang="pl" />
-    <file path="strings/brave_components_resources_pt-BR.xtb" lang="pt-BR" />
-    <file path="strings/brave_components_resources_pt-PT.xtb" lang="pt-PT" />
-    <file path="strings/brave_components_resources_ro.xtb" lang="ro" />
-    <file path="strings/brave_components_resources_ru.xtb" lang="ru" />
-    <file path="strings/brave_components_resources_sk.xtb" lang="sk" />
-    <file path="strings/brave_components_resources_sl.xtb" lang="sl" />
-    <file path="strings/brave_components_resources_sr.xtb" lang="sr" />
-    <file path="strings/brave_components_resources_sv.xtb" lang="sv" />
-    <file path="strings/brave_components_resources_sw.xtb" lang="sw" />
-    <file path="strings/brave_components_resources_ta.xtb" lang="ta" />
-    <file path="strings/brave_components_resources_te.xtb" lang="te" />
-    <file path="strings/brave_components_resources_th.xtb" lang="th" />
-    <file path="strings/brave_components_resources_tr.xtb" lang="tr" />
-    <file path="strings/brave_components_resources_uk.xtb" lang="uk" />
-    <file path="strings/brave_components_resources_vi.xtb" lang="vi" />
-    <file path="strings/brave_components_resources_zh-CN.xtb" lang="zh-CN" />
-    <file path="strings/brave_components_resources_zh-TW.xtb" lang="zh-TW" />
+    <file path="strings/brave_components_strings_am.xtb" lang="am" />
```

Comment:
why  are you renaming all of these files? While they probably should be `*_strings` instead of `*_resources`, I think that is just confusing what is going on in this PR and it should be separate

Reply:
generate_embedded_i18n expects that xtb files have same name prefix as initial .grd file. https://source.chromium.org/chromium/chromium/src/+/main:base/win/embedded_i18n/generate_embedded_i18n.gni;l=129?q=generate_embedded_i18n&ss=chromium%2Fchromium%2Fsrc 
This is also visible by others grd/xtb files in the Chromium repo